dashboard/app/views/levels/show.html.haml

```diff
@@ -34,7 +34,8 @@
     = render partial: 'levels/widget'
   - else
     = render partial: "levels/#{@level.class.to_s.underscore}"
-  = render partial: 'levels/reference_area'
+  - unless @level.is_a?(StandaloneVideo)
+    = render partial: 'levels/reference_area'
```

Comment:
? there is: https://github.com/code-dot-org/code-dot-org/blob/standalone-video-level-type/dashboard/app/views/levels/_standalone_video.html.haml